# DP: Fix up omp.h for multilibs.

2008-06-09  Jakub Jelinek  <jakub@redhat.com>

	* omp.h.in (omp_nest_lock_t): Fix up for Linux multilibs.

--- a/src/libgomp/omp.h.in
+++ b/src/libgomp/omp.h.in
@@ -39,8 +39,8 @@
 
 typedef struct
 {
-  unsigned char _x[@OMP_NEST_LOCK_SIZE@] 
-    __attribute__((__aligned__(@OMP_NEST_LOCK_ALIGN@)));
+  unsigned char _x[8 + sizeof (void *)] 
+    __attribute__((__aligned__(sizeof (void *))));
 } omp_nest_lock_t;
 #endif
 
